packages/dashboard/src/DashboardLayout.tsx

```diff
@@ -121,17 +124,21 @@ export function DashboardLayout({
 
         // Props supplied by GoldenLayout
         // eslint-disable-next-line react/prop-types
-        const { glContainer, glEventHub } = props;
+        const { dh, glContainer, glEventHub } = props;
         return (
-          <Provider store={store}>
-            <PanelErrorBoundary
-              glContainer={glContainer}
-              glEventHub={glEventHub}
-            >
-              {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-              <CType {...props} ref={ref} />
-            </PanelErrorBoundary>
-          </Provider>
+          // Enterprise should be able to override the JSAPI
+          // for each panel via the props
+          <ApiContext.Provider value={dh ?? defaultDh}>
```

Comment:
`dh` is already `DhType`, if we need null-coalescing here, then the prop should be `dh?: DhType` likely instead. Or even just `PanelProps`, which is correctly typed.